lib/external_service_new/shopify_dispatcher.rb

```diff
@@ -5,6 +5,8 @@
 module ExternalServiceNew
   class ShopifyDispatcher < RestClientDispatcher
 
+    class PaymentRequired < RuntimeError; end
```

Comment:
Isn't that class generic? It should not live within the `ShopifyDispatcher`, else the caller will have to rescue `ShopifyDispatcher::PaymentRequired`, which is what I wanted to move away from.

Besides, if a new connector requires payment, will you add a `MagentoDispatcher::PaymentRequired` class? And rescue the 2<sup>nd</sup> exception in calling code?